assets/src/design-system/components/search/input/components.js

```diff
@@ -136,9 +132,9 @@ const BaseDecorationContainer = css`
 export const ClearButton = styled.button`
   ${BaseDecorationContainer};
   display: ${({ isVisible }) => (isVisible ? 'flex' : 'none')};
-
+  outline: none;
   ${({ theme }) => themeHelpers.focusableOutlineCSS(theme.colors.border.focus)};
-  height: 32px;
+  height: 30px;
```

Comment:
This is so that when a keyboard focus occurs the focus looks clean and tight against the input.